client/packages/host/src/PluginsProvider.tsx

```diff
@@ -19,8 +19,19 @@ const getPluginArea = (area: string) => {
 
 const getPluginType = (type: string) => {
   switch (type) {
+    case 'dashboard':
```

Comment:
```suggestion
    case 'Dashboard':
```

This wasn't working for me...

Reply:
yes! sorry - had a note to update that and haven't done it yet